### PR TITLE
controls: document 'orientation_center/_left/_right'

### PR DESCRIPTION
**Pull request by ronso0**

### CODE REVIEW

**Comment by Holzhaus** [comment]
Partially fixes #261

File: source/chapters/advanced_topics.rst [hunk 1]
```diff
@@ -1675,6 +1675,27 @@ Any control listed above for :mixxx:cogroupref:`[ChannelN]` will work for a samp
    .. versionadded:: 1.9.0
 
 
+.. mixxx:control:: [ChannelN],orientation_center
+                   [PreviewDeckN],orientation_center
+                   [SamplerN],orientation_center
```
Comment (Holzhaus) [inline source/chapters/advanced_topics.rst:1680]: [AuxiliaryN], too

**Review by Holzhaus — COMMENTED** [review]
Thanks

File: source/chapters/advanced_topics.rst (hunk 1 above)
Comment (ronso0) [reply source/chapters/advanced_topics.rst:1680]: yep, good catch. fixed

**Comment by ronso0** [comment]
Ready.

**Review by Holzhaus — APPROVED** [review]
Thanks, LGTM.

**Pull request merged**